docs/integrations/language-clients/java/client/_snippets/_v0_8.mdx

```diff
@@ -659,3 +664,148 @@ Complete examples code is stored in the repo in a 'example` [folder](https://git
 - [client-v2](https://github.com/ClickHouse/clickhouse-java/tree/main/examples/client-v2) - main set of examples.
 - [demo-service](https://github.com/ClickHouse/clickhouse-java/tree/main/examples/demo-service) - example of how to use the client in a Spring Boot application.
 - [demo-kotlin-service](https://github.com/ClickHouse/clickhouse-java/tree/main/examples/demo-kotlin-service) - example of how to use the client in Ktor (Kotlin) application.
+
+## Migration From V1 ( =< 0.7.x ) {#migration_from_v1}
+
+
+Old client (V1) was using `com.clickhouse.client.ClickHouseClient#builder` as start point. The new client (V2) uses similar pattern with `com.clickhouse.client.api.Client.Builder`. Main
+differences are:
+- no service loader is used to grab implementation. The `com.clickhouse.client.api.Client` is facade class for all kinds of implementation in the future.
+- a fewer sources of configuration: one is provided to the builder and one is with operation settings (`QuerySettings`, `InsertSettings`). Previous version had configuration per node and was loading
+env. variables in some cases.
+
+### Configuration Parameters Match {#migration_from_v1_config}
+
+There are 3 enum classes related to configuration in V1:
+- `com.clickhouse.client.config.ClickHouseDefaults` - configuration parameters that supposed to be set in most use cases. Like `USER` and `PASSWORD`.
+- `com.clickhouse.client.config.ClickHouseClientOption` - configuration parameters specific for the client. Like `HEALTH_CHECK_INTERVAL`.
+- `com.clickhouse.client.http.config.ClickHouseHttpOption` - configuration parameters specific for HTTP interface. Like `RECEIVE_QUERY_PROGRESS`.
+
+They were designed to group parameters and provide clear separation. However in some cases it lead to a confusion (is there a difference between `com.clickhouse.client.config.ClickHouseDefaults#ASYNC` and
+`com.clickhouse.client.config.ClickHouseClientOption#ASYNC`). The new V2 client uses `com.clickhouse.client.api.Client.Builder` as single dictionary of all possible client configuration options.There is
+`com.clickhouse.client.api.ClientConfigProperties` where all configuration parameter names are listed.
+
+Table below shows what old options are supported in the new client and there new meaning.
+✔	- supported
+✗ - dropped
+| V1 Configuration | Matching V2 Builder Method | Comments          |
```

Comment:
I wonder if there is a nicer way to display this, the scroll is hard. @Blargian do you have any ideas of what we could do here?

@chernser is there a way to group them and break this up into digestible chunks/smaller tables to make it a bit more readable 🤔

Reply:
+1 to collocate them in groups.

If the page is large, we can consider moving it to a dedicated page